packages/randoml/randoml.0.1.4/opam

```diff
@@ -0,0 +1,41 @@
+opam-version: "2.0"
+synopsis: "Generating cryptographically-secure random numbers"
+description:
+  "A library relying on Rust rand to generate random numbers. The random numbers are cryptographically secure."
+maintainer: ["david@o1labs.org"]
+authors: ["David Wong"]
+license: "ISC"
+homepage: "https://github.com/mimoo/randoml"
+bug-reports: "https://github.com/mimoo/randoml/issues"
+depends: [
+  "conf-rust-2018"
+  "bignum" {>= "v0.14.0"}
+  "dune" {>= "2.8.0"}
+  "ppx_inline_test" {with-test & >= "v0.14.1"}
+  "core_kernel" {> "v0.14.0"}
+  "odoc" {with-doc}
+]
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+]
+dev-repo: "git+https://github.com/mimoo/randoml.git"
```

Comment:
```suggestion
dev-repo: "git+https://github.com/mimoo/randoml.git"
post-messages: [
  "This package requires at least rust version 1.53 to compile. Please make sure your system has a compatible version installed and try again." {failure}
]
```

I think this is the most convenient solution at the moment. This message will appear if the package fails to compile, which most likely will depend on the too old rust version installed.